packages/Uploader/src/components/DropZone/DropZone.styles.js

```diff
@@ -12,24 +13,35 @@ export const DropZone = styled.div(
     display: flex;
     justify-content: center;
     padding: ${tokens.space};
-
+    text-align: center;
     ${isDraggingOver &&
       css`
         background-color: ${tokens.color.blackLighten80};
         border: 2px solid ${tokens.color.purpleDarken10};
         color: ${tokens.color.purpleDarken10};
       `}
-
-    span[data-pka-anchor='button'] {
-      ${stylers.fontSize()};
-      margin-top: -${tokens.spaceSm};
-      min-height: auto;
-      padding: 0;
-    }
   `
 );
 
+export const DropZoneButton = styled(Button)`
+  ${stylers.fontSize()};
+  margin-top: -${tokens.spaceSm};
+  min-height: auto;
+  padding: 0;
+`;
+
 export const Body = styled.div`
   display: inline;
   margin-left: ${tokens.space};
 `;
+
+export const DropZoneIcon = styled.span`
+  flex-shrink: 0;
+`;
+
+export const Input = styled.input`
+  ${stylers.visuallyHidden};
+  &:focus + div [role="button"] {
```

Comment:
small comment, the space here looks like an extra space if someone doesn't look into the html code, maybe make it more explicit like `&:focus + div span[role="button"]`